libs/components/packages/src/schematics/migrations/update-9/legacy-services/legacy-services.schematic.spec.ts

```diff
@@ -0,0 +1,125 @@
+import { SchematicTestRunner } from '@angular-devkit/schematics/testing';
+
+import { createTestApp } from '../../../testing/scaffold';
+
+describe('legacy-services.schematic', () => {
+  const runner = new SchematicTestRunner(
+    'schematics',
+    require.resolve('../../migration-collection.json')
+  );
+
+  async function setupTest(options: { textContent: string }) {
+    const tree = await createTestApp(runner, {
+      projectName: 'my-app',
+    });
+
+    tree.overwrite('./src/app/app.component.ts', options.textContent);
+
+    return {
+      runSchematic: () => runner.runSchematic('legacy-services', {}, tree),
+      tree,
+    };
+  }
+
+  it('should replace services with legacy versions', async () => {
+    const { runSchematic } = await setupTest({
+      textContent: `import {SkyModalService} from '@skyux/modals';
```

Comment:
Should there be an example of multiple imports from `@skyux/modals` instead of just `SkyModalService`? I could see a regex getting tripped up on that.

```
import { SkyModalService, SkyModalConfiguration } from '@skyux/modals';
```

Reply:
Good call. Since we're using AST it shouldn't have trouble finding the `SkyModalService` identifier, but I updated the test to be more specific just so we're sure.